feat(docs): include output preview in the outputs doc (#1026)
kestra-io/docs: content/docs/06.workflow-components/07.outputs.md

2. once to access the outputs of the subflow itself — specifically, the `final` output.
::

Here is what you will see in the **Outputs** tab of the **Executions** page in the parent flow:

![subflow_output_parent](/docs/workflow-components/subflow_output_parent.png)

## Using Render Expression ##

You can evaluate the output further using the **Render Expression** functionality in the **Outputs** tab. Consider the following flow:

```yaml
id: json_values
namespace: dev
tasks:
- id: sample_json
  type: io.kestra.core.tasks.debugs.Return
  format: '{"data": [1, 2, 3]}'
```

> On running this flow, the **Outputs** tab would have the output for the `sample_json` task as shown.

Reviewer: ```suggestion
When you run this flow, the **Outputs** tab will contain the output for the `sample_json` task, as shown below:
```

![json_values](/docs/workflow-components/outputs/json_values.png)

> You can select the task from the drop-down, in this case we will select "sample_json", and click on **Render expression**.

Reviewer: ```suggestion
You can select the task from the drop-down menu. Here, we will select "sample_json" and click on **Render expression**:
```
